### ISSUE TITLE
centrepompidou images

### ISSUE THREAD

**Issue by emallen1973** [lovasoa/dezoomify#690]
I am trying to dezoomify images from centrepompidou, like:

https://www.centrepompidou.fr/es/ressources/oeuvre/cknG7R

The usual process does not work

**Comment by agmmnn** [comment]
you can download it with using dezoomify-rs:

```
.\dezoomify-rs "https://www.centrepompidou.fr/media/picture/aa/ec/aaec1c59e2a10192275338fbfaf760d6/dzi/uhd_files/12/{{x}}_{{y}}.jpg"
```

https://github.com/lovasoa/dezoomify-rs/wiki/Usage-example-for-the-custom-YAML-dezoomer#stage-2-construct-the-url-template

**Comment by emallen1973** [comment]
Thank you so much for your help. I was able to download them.

**Comment by emallen1973** [comment]
Hello Gökçe:

I need help figuring out how to dezoomify images from the Pushkin Museum
like:

https://collection.pushkinmuseum.art/en/entity/OBJECT/77609?query=Picasso&index=5

Thanks.

Enrique

**Comment by agmmnn** [comment]
Hey, this site and another site using the same zoomify algorithm have already been discussed under this issue. https://github.com/lovasoa/dezoomify/issues/651#issuecomment-1374544942

It's not supported by Dezoomify, but it doesn't seem that difficult to support this zoomify algorithm. If I have time, I can write a script myself.

**Comment by agmmnn** [comment]
hey can you try this app, https://github.com/agmmnn/pushkin-downloader

**Comment by lovasoa** [comment]
Would you be interested to include your implementation of the algorithm into dezoomify ? It should be a simple .js file to create.

**Comment by emallen1973** [comment]
Hi Gökçe,

For some reason, a couple of the dezoomified images appear deformed:

https://collection.pushkinmuseum.art/entity/OBJECT/77606?index=8&paginator=entity-set&entityType=PERSON&entityId=260&attribute=objects

and

https://collection.pushkinmuseum.art/en/entity/OBJECT/77613?query=Picasso&index=7

It looks like some of the pieces are placed incorrectly when the image
is rebuilt.

**Comment by agmmnn** [comment]
Yes I am aware of this too, corner tiles have this problem. I will try to implement this code in dezoomify and I think I will fix this problem in the meantime.

**Comment by clmntgbrl** [comment]
> you can download it with using dezoomify-rs:
> 
> ```
> .\dezoomify-rs "https://www.centrepompidou.fr/media/picture/aa/ec/aaec1c59e2a10192275338fbfaf760d6/dzi/uhd_files/12/{{x}}_{{y}}.jpg"
> ```
> 
> https://github.com/lovasoa/dezoomify-rs/wiki/Usage-example-for-the-custom-YAML-dezoomer#stage-2-construct-the-url-template

I'm trying to download from centrepompidou as well and can't even find the link to jpgs in the inspector. Can anybody help ? (btw thanks for the extension, it really is great !)

**Comment by agmmnn** [comment]
@clmntgbrl 
zoom all the way into the picture, you will see the last zoom level.

![KLaEua2Ysh](https://user-images.githubusercontent.com/16024979/224542930-97f7ce7d-af34-475d-a4f1-f9acc6623954.png)

**Comment by clmntgbrl** [comment]
many, many thanks, worked like a charm !

**Comment by lovasoa** [comment]
Closing as a duplicate site report for centrepompidou.fr. The oldest open tracking issue I found for this website is #666; please keep follow-up investigation for this site there.
